.github/workflows/release.latest.yml

```diff
@@ -0,0 +1,21 @@
+name: release-latest
+on:
+  push:
+    tags:
+      - '**'
```

Comment:
What does `**` do more than `*`?

Reply:
`**` matches everything vs `*` which only matches strings that do not contain a slash (`/`).

Ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags